runtime/moonbase/tests/xcm_mock/statemint_like.rs

```diff
@@ -315,14 +315,30 @@ impl<AssetLocation: Get<Location>> ContainsPair<Asset, Location>
 
 pub type TrustedTeleporters = (ConcreteAssetFromRelay<RelayTokenLocation>,);
 
+/// Trust reserves from sibling parachains, relay chain, and here
+pub struct TrustedReserves;
+impl ContainsPair<Asset, Location> for TrustedReserves {
+	fn contains(asset: &Asset, origin: &Location) -> bool {
+		let AssetId(location) = &asset.id;
+		match (&asset.id.0.parent_count(), &asset.id.0.first_interior()) {
+			// Sibling parachains
+			(1, Some(Junction::Parachain(_id))) => location.chain_location() == *origin,
+			// Relay chain
+			(1, _) => location.chain_location() == *origin,
+			// Here
+			(0, _) => Location::here() == *origin,
+			_ => false,
+		}
+	}
+}
+
 pub struct XcmConfig;
 impl Config for XcmConfig {
 	type RuntimeCall = RuntimeCall;
 	type XcmSender = XcmRouter;
 	type AssetTransactor = AssetTransactors;
 	type OriginConverter = XcmOriginToTransactDispatchOrigin;
-	type IsReserve =
-		orml_xcm_support::MultiNativeAsset<orml_traits::location::RelativeReserveProvider>;
+	type IsReserve = TrustedReserves;
```

Comment:
Is there a specific reason why we use `TrustedReserves` struct here instead of the one used in this other xcm_mock for statemint in moonbeam runtime? https://github.com/moonbeam-foundation/moonbeam/pull/2969/files#diff-d6d92b6a499a1e22922240143cd564e66a52a71f27d7b5225ea6a30786e1167cR326.

If there is no specific reason for this custom `TrustedReserves` struct here, I would suggest to keep the same config as we have in `runtime/moonbeam/tests/xcm_mock/statemint_like.rs` to be consistent between different mocks.

Reply:
Removed the `TrustedReserves` to keep the same config in different mocks